examples/ping_pong.rs

```diff
@@ -155,13 +164,37 @@ async fn main() -> Result<(), Error> {
     let mut input = String::new();
     let _ = io::stdin().read_line(&mut input)?;
 
-    let ice_agent = Arc::new(
-        Agent::new(AgentConfig {
-            network_types: vec![NetworkType::Udp4],
-            ..Default::default()
-        })
-        .await?,
-    );
+    let udp_network = if use_mux {
+        use ice::udp_mux::*;
+        let port = if is_controlling { 4000 } else { 4001 };
+
+        let udp_socket = UdpSocket::bind(("0.0.0.0", port)).await?;
+        let udp_mux = UDPMuxDefault::new(UDPMuxParams::new(udp_socket));
+
+        UDPNetwork::Muxed(udp_mux)
+    } else {
+        UDPNetwork::Ephemeral(Default::default())
+    };
+
+    let ice_agent = if is_controlling {
```

Comment:
what's the difference between if and else?

Reply:
Good catch, there is none. I moved this above.